lib/stripe/api.ex

```diff
@@ -273,4 +275,15 @@ defmodule Stripe.API do
       _ -> body
     end
   end
+
+  defp add_object_expansion(url, expansion) when is_list(expansion) do
+    expand_str =
+      expansion
+      |> Enum.map(&"expand[]=#{&1}")
+      |> Enum.join("&")
+
+    "#{url}?#{expand_str}"
+  end
+
```

Comment:
could remove space and/or put above first function if you wanted.

Reply:
Ah, these spaces are being added by the formatter. Is this something you would prefer I bypass in this case or are you working towards formatting the code base?

Reply:
na no worries :)